README.md

```diff
@@ -1,21 +1,15 @@
-![Electron Installer for Debian](resources/logo.png)
+![Electron Installer for Flatpak](resources/logo.png)
 
-# grunt-electron-installer-debian [![Version](https://img.shields.io/npm/v/grunt-electron-installer-debian.svg)](https://www.npmjs.com/package/grunt-electron-installer-debian) [![Build Status](https://img.shields.io/travis/unindented/grunt-electron-installer-debian.svg)](http://travis-ci.org/unindented/grunt-electron-installer-debian) [![Dependency Status](https://img.shields.io/gemnasium/unindented/grunt-electron-installer-debian.svg)](https://gemnasium.com/unindented/grunt-electron-installer-debian)
+# grunt-electron-installer-flatpak [![Version](https://img.shields.io/npm/v/grunt-electron-installer-flatpak.svg)](https://www.npmjs.com/package/grunt-electron-installer-flatpak) [![Build Status](https://img.shields.io/travis/unindented/grunt-electron-installer-flatpak.svg)](http://travis-ci.org/unindented/grunt-electron-installer-flatpak) [![Dependency Status](https://img.shields.io/gemnasium/unindented/grunt-electron-installer-flatpak.svg)](https://gemnasium.com/unindented/grunt-electron-installer-flatpak)
```

Comment:
Yeah, I'll update this, but this whole status line is pretty much broken.

Could definitely rewrite the tests and integrate with travis (can pull flatpak in from a ppa on travis I'm pretty sure), and then figure out what the hell gemnasium is, but was planning to leave testing and ci for a follow up of the initial pass at this work.

I'll open up an issue.